Modelica/Electrical/Machines/Examples/SynchronousMachines/SMPM_NoLoad.mo

```diff
@@ -74,9 +74,10 @@ equation
     annotation (Line(points={{-20,10},{-20,12},{-10,12}},color={0,0,255}));
   connect(potentialSensor.plug_p, terminalBox.plugSupply)
     annotation (Line(points={{0,20},{0,12}}, color={0,0,255}));
-  connect(potentialSensor.phi, toSpacePhasor.u) annotation (Line(points={{8.88178e-16,
-          41},{8.88178e-16,50},{8,50}}, color={0,0,127}));
-  connect(speed.flange, smpm.flange) annotation (Line(points={{30,0},{10,0}}));
+  connect(potentialSensor.phi, toSpacePhasor.u) annotation (Line(points={{0,
+          41},{0,50},{8,50}}, color={0,0,127}));
+  connect(constantSpeed.flange, smpm.flange)
```

Comment:
This reverted the change from #4031 where `constantSpeed` was renamed to `speed`, which breaks the `SMPM_NoLoad` model.

Reply:
Indeed. Addressed via #4146. Thanks for reporting.